README.md

```diff
@@ -190,4 +190,4 @@ This project has adopted the Google Open Source Code of Conduct. Please see [cod
 
 ### License
 
-The Dataflux Python Client has an Apache License 2.0. Please see the [LICENSE](LICENSE) file for more information.
\ No newline at end of file
+The Dataflux Python Client has an Apache License 2.0. Please see the [LICENSE](LICENSE) file for more information.
```

Comment:
Oh I think I made a typo on "Dataflux Python Client" originally. Could you change the naming there?

Reply:
Done